@here/harp-mapview/lib/Utils.ts

```diff
@@ -239,9 +239,143 @@ export namespace MapViewUtils {
             groundPlane.constant = 0;
             groundSphere.radius = EarthConstants.EQUATORIAL_RADIUS;
         }
+        if (result !== null) {
```

Comment:
Please change the signature (i.e. return value type) of the function. It should never return null now

Reply:
As I mentioned above, just for debugging, normally I could put assert here, but wanted to test drive the stuff - of course not to merge yet.